loader/loader.c

```diff
@@ -2161,6 +2162,44 @@ static VkResult loader_read_layer_json(const struct loader_instance *inst, struc
         }
     }
 
+    required_instance_extensions = cJSON_GetObjectItem(layer_node, "required_instance_extensions");
+    if (NULL != required_instance_extensions) {
+        if (!loader_check_version_meets_required(loader_combine_version(1, 3, 0), version)) {
```

Comment:
Adding a next layer manifest version is fine, it just need to change the `docs/LoaderLayerInterface.md` that a new version has been added with what changed.